### PR TITLE
 net-news/rssguard: version bump to 3.5.5 and live version update

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: net-news/rssguard/rssguard-9999.ebuild [hunk 1]
```diff
@@ -3,41 +3,36 @@
 
 EAPI=6
 
-inherit git-r3 qmake-utils
+inherit flag-o-matic git-r3 qmake-utils
 
 DESCRIPTION="A tiny RSS and Atom feed reader"
 HOMEPAGE="https://github.com/martinrotter/rssguard"
-EGIT_REPO_URI="https://github.com/martinrotter/${PN}.git"
+EGIT_REPO_URI="https://github.com/martinrotter/rssguard.git"
```
Comment (a17r) [inline net-news/rssguard/rssguard-9999.ebuild]: Why this change?

Comment (soredake) [reply net-news/rssguard/rssguard-9999.ebuild]: https://devmanual.gentoo.org/ebuild-writing/variables/
```
The examples of bad constant-value variable references are:

using ${PN} in HOMEPAGE, EGIT_REPO_URI or the domain of SRC_URI — it breaks URL parsing in editors and terminals, and makes it hard to copy the link for external use (e.g. while reviewing via gitweb), 
```

Comment (a17r) [reply net-news/rssguard/rssguard-9999.ebuild]: Thanks, that I can accept.

**Comment by soredake** [comment]
@a17r reported to upstream https://github.com/martinrotter/rssguard/issues/157 and updated 2016 to 2017.

**Comment by soredake** [comment]
RCC errors is fixed in master and 9999 ebuild.

**Comment by soredake** [comment]
@a17r all errors is fixed and ebuild updated to 3.5.5.

**Comment by a17r** [comment]
lrelease is missing `|| die`, but will fix on merge
